utils/collectd/Makefile

```diff
@@ -335,7 +335,7 @@ $(eval $(call BuildPlugin,netlink,netlink input,netlink,+PACKAGE_collectd-mod-ne
 $(eval $(call BuildPlugin,network,network input/output,network,+PACKAGE_COLLECTD_ENCRYPTED_NETWORK:libgcrypt))
 $(eval $(call BuildPlugin,nginx,nginx status input,nginx,+PACKAGE_collectd-mod-nginx:libcurl))
 $(eval $(call BuildPlugin,ntpd,NTP daemon status input,ntpd,))
-$(eval $(call BuildPlugin,nut,UPS monitoring input,nut,+PACKAGE_collectd-mod-nut:nut))
+$(eval $(call BuildPlugin,nut,UPS monitoring input,nut,+PACKAGE_collectd-mod-nut:nut-common))
```

Comment:
On Mon, 2016-07-04 at 05:40 -0700, champtar wrote:
> input,nut,+PACKAGE_collectd-mod-nut:nut-common))
> you may need to bump collectd PKG_RELEASE

In fact, yes, I missed that.  Thanks.

Regards,

Daniel